.vscode/launch.json

```diff
@@ -126,6 +126,6 @@
       "webRoot": "${workspaceRoot}",
       "preLaunchTask": "Debug: Word Desktop",
       "postDebugTask": "Stop Debug"
-    }
+    },
```

Comment:
Note: an alternative to using regex to modify this file in the convert script is we could make this file valid json and modify it as a json object instead (the tasks.json file is being modified this way).  Doing that would mean having to leave the comma off the end and removing the comments at the top of the file (VS Code doesn't require the file to be valid json despite the file extension).

I don't have a personal preference for which method to use . . . just pointing out the two alternatives (should probably standardized which way is used in the convert script).

Reply:
> Note: an alternative to using regex to modify this file in the convert script is we could make this file valid json and modify it as a json object instead (the tasks.json file is being modified this way). Doing that would mean having to leave the comma off the end and removing the comments at the top of the file (VS Code doesn't require the file to be valid json despite the file extension).
> 
> I don't have a personal preference for which method to use . . . just pointing out the two alternatives (should probably standardized which way is used in the convert script).

I favor making it valid JSON. It will be easier to edit than working with RegEx. Also, its misleading to have a "json" extension on a file that isn't valid "json".

Reply:
Made it valid JSON